src/storage/addtoCard.js

```diff
@@ -88,6 +92,8 @@ const AddtoCardCtxProvider = ({ children }) => {
     cardProducts: addedProduct,
     productCountReSet: reSetProductCount,
     countProductChange: countProductChange,
+    setTotalCounte: setTotalCounteHandler,
```

Comment:
in az biroon nabayd dade beshe ke, in ye fielde mohasebati hastesh, va aslan state nemikhad.
kollan `productCount` inam state nemikhad:
`totalCount: addedProduct.reduce((a, product) => a + product.count, 0)`